tutor/src/helpers/plan.coffee

```diff
@@ -27,4 +27,9 @@ PlanHelper =
     {isPublishing, publishStatus}
 
 
+  isPlanOpen: (plan) ->
+    now = moment(TimeStore.getNow())
+    !! _.find plan?.tasking_plans, (tasking) -> moment(tasking.opens_at).isBefore(now)
```

Comment:
`isPlanOpen`/something similar does exist.  It's just my fault that it's in a weird place [in the `PlanMixin`](https://github.com/openstax/tutor-js/blob/master/tutor/src/components/task-plan/plan-mixin.cjsx#L25-L27).  This does get pulled in to the parents of the `PlanFooter`, so the `state.isVisibleToStudents`  could be passed in to `PlanFooter` and `DeleteLink` as a `prop`.